doc/guide/build_windows.hpp

```diff
@@ -0,0 +1,95 @@
+/*! @page build_windows Building mlpack From Source
+
+@section build_intro Introduction
+
+This document discusses how to build mlpack for Windows from source, so you can later
+create your own C++ applications.
+
+@section build_env Environment
+
+This tutorial has been designed and tested using:
+- Windows 10
+- Visual Studio 2017 (toolset v141)
+- mlpack-3.0.2
+- OpenBLAS.0.2.14.1
+- boost_1_66_0-msvc-14.1-64
+- armadillo-8.500.1
+- and x64 configuration
+
+The directories and paths used in this tutorial are just for reference purposes.
+
+@section build_prereqs Pre-requisites
+
+- Install CMake for Windows (win64-x64 version from https://cmake.org/download/)
+and make sure you can use it from the Command Prompt (may need to add to the PATH)
+
+- Download the latest mlpack release from here:
+<a href="http://www.mlpack.org/files/mlpack-3.0.2.tar.gz">mlpack-3.0.2</a>
```

Comment:
I was thinking if it might be useful to provide an alias for the latest package so that we don't have to update this tutorial once we have a new release. @rcurtin what do you think?

Reply:
I already go through `doc/` when I do every release and increment any version numbers, so it's not a huge issue either way.  Aliases are nice if we can do it; do you have a suggested way to do it?